lib/matplotlib/animation.py

```diff
@@ -386,16 +400,21 @@ def isAvailable(cls):
         if not bin_path:
             return False
         try:
-            p = subprocess.Popen(bin_path,
-                             shell=False,
-                             stdout=subprocess.PIPE,
-                             stderr=subprocess.PIPE,
-                             creationflags=subprocess_creation_flags)
-            p.communicate()
-            return True
+            p = subprocess.Popen(
+                bin_path,
+                shell=False,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                creationflags=subprocess_creation_flags)
+            return cls._handle_subprocess(p)
         except OSError:
             return False
 
+    @classmethod
```

Comment:
Is there a reason for using @classmethod rather than @staticmethod here, even though the method doesn't use its `cls` argument?